src/shapes/Text/Text.ts

```diff
@@ -818,12 +829,13 @@ export class Text<
         // we can measure the kerning couple and subtract the width of the previous character
         coupleWidth = ctx.measureText(couple).width;
         fontCache[couple] = coupleWidth;
-        kernedWidth = coupleWidth - previousWidth;
+        // safe to use the non-null since if undefined we defined it before.
```

Comment:
this belongs on line 838 I think

Reply:
I m commenting on the `!` for ` previousWidth`